app/src/App.tsx

```diff
@@ -0,0 +1,69 @@
+import axios from "axios";
+import { useState } from "react";
+import "./App.css";
+
+declare global {
+  interface Window {
+    Razorpay: any;
+  }
+}
+
+const App = () => {
+  const [shoe, setShoe] = useState({
+    name: "Training Shoes",
+    creator: "Nike",
+    img: "https://images.pexels.com/photos/3490360/pexels-photo-3490360.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500",
+    price: 500,
+  });
+
+  const initPay = (data: any) => {
```

Comment:
type this `data` to a custom type, we refrain from using `any`

Reply:
resolved, the payment gateway comes with built-in interface for this